Source/WebCore/rendering/RenderLayerBacking.cpp

```diff
@@ -3228,6 +3228,12 @@ bool RenderLayerBacking::isBitmapOnly() const
     return m_owningLayer.isBitmapOnly();
 }
 
+#if ENABLE(RE_DYNAMIC_CONTENT_SCALING)
+bool RenderLayerBacking::layerAllowsDynamicContentScaling(const GraphicsLayer*) const
+{
+    return m_owningLayer.allowsDynamicContentScaling();
```

Comment:
For WebGL/GPU, I think we usually end up with a 'contents' platform layer that holds the output frames directly, and then all the normal HTML content (borders, backgrounds etc) will be in a different platform layer.

I think this change will disable the dynamic content scaling for the background unnecessarily, and we could potentially fix that. 

Unsure if it matters though, since we also lose dynamic content scaling when the canvas contents get drawn into the same layer as the background (canvas 2d).

Reply:
I think "does not matter", since their contents will still not scale, just like 2D Canvas.

I did briefly consider doing something similar for 2D Canvas (contents in a different layer than decorations, to split them, so we could keep decorations scaling), but I don't think it's worth it.